test/routes/sessions.test.js

```diff
@@ -4,7 +4,7 @@ const P = require('bluebird');
 const nock = require('nock');
 const otplib = require('otplib');
 const request = require('../helpers/request');
-const ServiceUrlGetter = require('../../src/services/service-url-getter');
+const forestUrlGetter = require('../../src/utils/forest-url-getter');
```

Comment:
[THINKING] You might want to use the inject() methods here, knowing that the file might be deleted